### PR TITLE
Added little work to InventoryRewrite

### PR DESCRIPTION
**Pull request by WildCard65**
Added sample CSGO inv.
Added more recent tf2 sample json(incase it changed?)

### CODE REVIEW

**Comment by WildCard65** [comment]
Oh ya, also threw in syncing with master repo master branch

File: SteamTrade/Inventory/Inventory.cs [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+        {
+            Parse();
+        }
+
+        public async Task FetchInventoryAsync(OnInventoryLoaded callback)
+        {
+            await ParseAsync(callback);
+        }
+
+#warning "This non-async method needs debugging as something is horrible with it."
```
Comment (WildCard65) [inline SteamTrade/Inventory/Inventory.cs]: @BlueRaja this is here until I can fix the Parse() method... The ParseAsync() method works perfectly and Parse() is a copy of it.

**Comment by WildCard65** [comment]
I say I'm done porting steambot to work with the new inv system. It does 100% break backward compatibility but as you said, it should be acceptable.

File: SteamTrade/Inventory/Inventory.cs (hunk 1 above)
Comment (BlueRaja) [inline SteamTrade/Inventory/Inventory.cs]: Why not just do `Task.Wait(ParseAsync)`

Comment (WildCard65) [inline SteamTrade/Inventory/Inventory.cs]: Instead of a Parse method()? Ok.

**Comment by WildCard65** [comment]
Ok, after these last few commits I think this is ready for a full review.

**Comment by BlueRaja** [comment]
This is impossible for me to review.  Remind me to sync this branch to develop tonight, then I will close this PR and ask you to recreate it.

**Comment by WildCard65** [comment]
Ok

**Pull request closed without merging**